docs/turbo-modules-xplat.md

```diff
@@ -52,7 +52,7 @@ CxxTurboModulesGuide
 
 ## 1. JavaScript Specification
 
-Create the following spec inside the `tm` folder:
+Create the following spec inside the `tm` folder (`NativeSampleModule.ts|js`):
```

Comment:
This information is already visible in the `title=` belows

Reply:
FWIW, I also found this confusing when reading the C++ TurboModule guide in order to learn how they work.

It doesn't seems like the `title=` filename information shows up when viewing through github.
![image](https://github.com/user-attachments/assets/c8abfdb6-78af-4abc-a3fd-3c174fedd298)